src/var/isFunction.js

```diff
@@ -0,0 +1,13 @@
+define( function() {
+	"use strict";
+
+	return function isFunction( obj ) {
+
+        // Support: Chrome <=57, Firefox <=52
```

Comment:
Do we even need this function anymore? Currently we're at Chrome 62 and Firefox 57.  **Edit:** Well hecky-darn nevermind, looks like Firefox ESR is [still on 52](https://www.mozilla.org/en-US/firefox/organizations/) until the middle of next year. 😢